examples/media/schema.json

```diff
@@ -29,13 +28,13 @@
               "type": "object",
               "oneOf": [
               {
-                "#ref": "medialibrary.videos.videoObject"
+                "#ref": "medialibrary.videos.<element>"
```

Comment:
Syntax "needs to be negotiated"... Shall "<bla>" become a valid object realtion in a schema?

Reply:
I thought, we might want to get rid of plural resource names and it 'duplicate' object names.. The name is derived anyway ;)

Videos -> video Object
Tracks-> trackObject 
etc.

Reply:
Anyway <element> is wrong at this place unless you want to use it as placeholder.